data/mods/Aftershock/items/comestibles/alienfood.json

```diff
@@ -132,5 +173,406 @@
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": "raw alien kidney",
+    "description": "You are guessing here, but this is likely the kidney of the animal.  The ammonia smell was the biggest hint.",
+    "color": "brown",
+    "healthy": -8,
+    "price_postapoc": 25,
+    "quench": -5,
+    "calories": 54,
+    "vitamins": [ [ "vitA", 6 ], [ "vitC", 2 ], [ "calcium", 0 ], [ "iron", 8 ], [ "vitB", 155 ], [ "mutant_toxin", 25 ] ]
+  },
+  {
+    "id": "alien_kidney_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_kidney",
+    "looks_like": "offal_cooked",
+    "name": "cooked alien kidney",
+    "description": "Thoroughly cooked and prepared, this alien organ has acquired the filthy ammonia smell of a poorly mopped spaceship deck.  Even if it were safe to eat, you are not sure you'd want to.",
+    "healthy": -2,
+    "price_postapoc": 25,
+    "quench": -2,
+    "fun": -3,
+    "parasites": 0,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_sweetbread",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": "raw alien sweetbread",
+    "description": "You got this chunk of meat from the throat of the animal; it looks different?",
+    "color": "brown",
+    "healthy": 0,
+    "price_postapoc": 25,
+    "quench": 0,
+    "calories": 133,
+    "vitamins": [ [ "vitA", 0 ], [ "vitC", 32 ], [ "calcium", 0 ], [ "iron", 6 ], [ "vitB", 20 ], [ "mutant_toxin", 25 ] ]
+  },
+  {
+    "id": "alien_sweetbread_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_sweetbread",
+    "looks_like": "meat_cooked",
+    "name": "cooked alien sweetbread",
+    "price_postapoc": 50,
+    "fun": -1,
+    "description": "Well, it's definitely not muscle, you can tell now.  It oozes a purple liquid out onto your hand as you hold it.",
+    "parasites": 0,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "alien_bone",
+    "category": "spare_parts",
+    "name": { "str": "alien bone" },
+    "weight": "225 g",
+    "color": "white",
+    "symbol": "%",
+    "description": "Obviously bones.  Could be used to make some stuff, like needles.",
+    "price": 0,
+    "price_postapoc": 10,
+    "material": [ "bone" ],
+    "milling": { "into": "meal_bone", "conversion_rate": 4 },
+    "volume": "250 ml",
+    "vitamins": [ [ "calcium", 96 ], [ "mutant_toxin", 25 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "alien_fat",
+    "name": { "str": "chunk of alien fat", "str_pl": "chunks of alien fat" },
+    "weight": "220 g",
+    "color": "pink",
+    "spoils_in": "7 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -10,
+    "healthy": -5,
+    "price_postapoc": 250,
+    "calories": 1923,
+    "description": "Freshly butchered fat from an alien.  You could eat it cooked or raw, but it is more safely used for crafting non edible-products.",
```

Comment:
Did fat's lards and tallows get added to appropriate recipes to make lamp oil and the like?

Reply:
Oh I haven't yet